apps/src/code-studio/progressRedux.js

```diff
@@ -375,22 +391,35 @@ const isCurrentLevel = (state, level) => {
  * about and (b) determines current status based on the current state of
  * state.levelProgress
  */
-const levelWithStatus = (state, level) => {
-  if (level.kind !== LevelKind.unplugged) {
-    if (!level.title || typeof(level.title) !== 'number') {
-      throw new Error('Expect all non-unplugged levels to have a numerical title');
+const levelWithStatus = (state, scriptLevel) => {
+  scriptLevel.levels.forEach(level => {
+    if (level.kind !== LevelKind.unplugged) {
+      if (!level.title || typeof(level.title) !== 'number') {
+        throw new Error('Expect all non-unplugged levels to have a numerical title');
+      }
+    }
+  });
+  let level;
+  if (scriptLevel.levels.some(level => level.ind === LevelKind.peer_review)) {
```

Comment:
Is suspect this should be `level.id` rather than `level.ind`. This also indicates it might be worth adding test coverage for some of these error conditions.